.github/renovate.json

```diff
@@ -1,6 +1,65 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:recommended"
-  ]
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "rebaseWhen": "behind-base-branch",
+    "labels": [
+        "dependencies"
+    ],
+    "extends": [
+        "config:recommended"
```

Comment:
@alcohol With this one it'll bring the default renovate rules for things like GitHub actions etc, kinda nice esp if this project doesn't have dependabot enabled.

Reply:
Yeah that's fine, there is no dependabot setup here if I am not mistaken.